pom.xml

```diff
@@ -22,7 +22,7 @@
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Hi, not happy with jdk 11 yet, there are heaps of places still on Java 8 including my workplace.